internal/authorization/schema.openfga

```diff
@@ -0,0 +1,16 @@
+model
+  schema 1.1
+type user
+type app
+type group
```

Comment:
It seems you are trying to model nested groups here.

A simpler/more performant way to do it would be 

```
type group
   relations
       define member : [user, group#member]
type app_group
    relations
       define member : [app, app_group#member]

If you want to define that the group:engineering is a child of group:everyone, you'd write a tuple

USER: group:engineering#member
RELATION: member
OBJECT: group:everyone

Reply:
Point taken, thanks for the suggestion